pkg/log/handler.go

```diff
@@ -0,0 +1,45 @@
+package log
+
+import (
+	"fmt"
+	"github.com/applike/gosoline/pkg/cfg"
+	"time"
+)
+
+type Handler interface {
+	Channels() []string
+	Level() int
+	Log(timestamp time.Time, level int, msg string, args []interface{}, err error, data Data) error
+}
+
+type HandlerFactory func(config cfg.Config, handlerIndex int) (Handler, error)
+
+var handlerFactories = map[string]HandlerFactory{
+	"iowriter": handlerIoWriterFactory,
```

Comment:
What other handler factories do you imagine to be implemented? Having the factory pattern for only one type looks a bit boilerplate-ish

Reply:
well, there are already the sentry and metric handlers..  
currently I have no plan for additional handlers but with this pattern it is open for extension..